pulp_rpm/app/models/advisory.py

```diff
@@ -100,7 +101,9 @@ class UpdateRecord(Content):
     # A field that represents the hash digest of the update record. Used to track differences
     # between two UpdateRecord objects without having to examine the associations like
     # UpdateCollection or UpdateCollectionPackage.
-    digest = models.TextField(unique=True)
+    digest = models.TextField(db_index=True)
+
+    _pulp_domain = models.ForeignKey("core.Domain", default=get_domain_pk, on_delete=models.PROTECT)
```

Comment:
I know it works both ways but is a direct import of a class an option here?  It feels more reliable in some abstract sense.

Reply:
I'm not sure what you're asking for here, sorry. This usage is just following the example of pulp_file.